examples/plot_inverse.py

```diff
@@ -141,3 +142,57 @@
 #
 # .. warning:: To use this pipeline, we need a cortical segmentation of MRI
 #   data, that could be provided by Freesurfer
+
+##############################################################################
+import pickle  # noqa
+from ephypype.gather.gather_results import get_results  # noqa
+from visbrain.objects import BrainObj, ColorbarObj, SceneObj  # noqa
+
+time_series_files, label_files = get_results(main_workflow.base_dir,
+                                             main_workflow.name,
+                                             pipeline='inverse')
+
+time_pts = 30
+
+sc = SceneObj(size=(800, 500), bgcolor=(.1, .1, .1))
+lh_file = op.join(subjects_dir, 'fsaverage', 'label/lh.aparc.annot')
+rh_file = op.join(subjects_dir, 'fsaverage', 'label/rh.aparc.annot')
+for inverse_file, label_file in zip(time_series_files, label_files):
+    # Load files :
+    with open(label_file, 'rb') as f:
+        ar = pickle.load(f)
+        names, xyz, colors = ar['ROI_names'], ar['ROI_coords'], ar['ROI_colors']  # noqa
+    ts = np.squeeze(np.load(inverse_file))
+    cen = np.array([k.mean(0) for k in xyz])
+
+    # Get the data of the left / right hemisphere :
+    lh_data, rh_data = ts[::2, time_pts], ts[1::2, time_pts]
+    clim = (ts[:, time_pts].min(), ts[:, time_pts].max())
+    roi_names = [k[0:-3] for k in np.array(names)[::2]]
+
+    # Left hemisphere outside :
+    b_obj_li = BrainObj('white', translucent=False, hemisphere='left')
+    b_obj_li.parcellize(lh_file, select=roi_names, data=lh_data, cmap='viridis')  # noqa
+    sc.add_to_subplot(b_obj_li, rotate='left')
+
+    # Left hemisphere inside :
+    b_obj_lo = BrainObj('white',  translucent=False, hemisphere='left')
+    b_obj_lo.parcellize(lh_file, select=roi_names, data=lh_data, cmap='viridis')  # noqa
+    sc.add_to_subplot(b_obj_lo, col=1, rotate='right')
+
+    # Right hemisphere outside :
+    b_obj_ro = BrainObj('white',  translucent=False, hemisphere='right')
+    b_obj_ro.parcellize(rh_file, select=roi_names, data=rh_data, cmap='viridis')  # noqa
+    sc.add_to_subplot(b_obj_ro, row=1, rotate='right')
+
+    # Right hemisphere inside :
+    b_obj_ri = BrainObj('white',  translucent=False, hemisphere='right')
+    b_obj_ri.parcellize(rh_file, select=roi_names, data=rh_data, cmap='viridis')  # noqa
+    sc.add_to_subplot(b_obj_ri, row=1, col=1, rotate='left')
+
+    # Add the colorbar :
+    cbar = ColorbarObj(b_obj_li, txtsz=15, cbtxtsz=20, txtcolor='white',
+                       cblabel='Intensity')
+    sc.add_to_subplot(cbar, col=2, row_span=2)
+
+sc.preview()
```

Comment:
it's a lot of code for an example! we should make it less so that beginners are not intimidated.

Reply:
I've suggested that you include a wrapper in neuropycon that uses Visbrain objects but make it easier for users to have their plot. @annapasca told me that this is something that we can add in the future but to have a quick draft, maybe we should keep it like that

Reply:
I agree! But do you have an idea what such a wrapper might look like in terms of API? Should this go into visbrain or neuropycon? Maybe we could have different nodes for different visbrain functionality -- like one for connectivity, one for power etc?

Reply:
Hi @EtienneCmb @jasmainak ! 
I came back to this PR, and about the wrapper, yes, I think it's something we can think in a second moment